src/api/db/models/Image.ts

```diff
@@ -144,23 +144,86 @@ export class ImageModel {
     context: Pick<Context, 'user'>,
   ): Promise<gql.StandardErrorPayload> {
     try {
-      const res = await Promise.allSettled(
-        input.imageIds!.map((imageId) => {
-          return this.deleteImage({ imageId }, context);
+      // Current limit of image deletion due to constraints of s3 deleteObjects
+      if (input.imageIds!.length > 300) {
```

Comment:
Failing here because if not we would delete the mongo records but not s3 objects

Reply:
Do we have a standard place for constants, looked in config and not sure it would be the same thing